models/namespace.js

```diff
@@ -82,7 +82,7 @@ export default {
   },
 
   projectNameSort() {
-    return this.project?.nameSort || '';
+    return this.project?.nameSort || '}';
```

Comment:
This was the remnants of a test. It shouldn't be here but it may be masking a sorting issue. So I'll remove it and fix the underlying issue if I run into it.

Reply:
I can't remember this from before, so it's either memory fail or a regression, but in the project/namespace list with grouping selected the 'Not in a Project' group is sorted alphabetically in line with project names. That's not something i'd assume given their labelling... `Project: a`, `Project: b`, `Not in a Project`, `Project: r`, etc. I created a couple of projects and thought you'd lost them due to this misunderstanding of the sort order.
  After playing around some more, when namespaces are added to the group they then appear above `Not in a Project`. So given the example above if project `r` was empty, creating/moving a namespace to it would result in the following order `Project: a`, `Project: b`, `Project: r`, Not in a Project`. 
  So I guess question is should the `Project` preffix  or the project content be taken into account when sorting by name?